src/screens/TopicPageScreen/elements/TopicDomainHeader.js

```diff
@@ -36,21 +43,42 @@ const TopicDomainHeader = (props) => {
             marginRight: normalize(5),
             height: normalize(8)
           }}
+          width={isPreview ? 9 : 16}
+          height={isPreview ? 9 : 16}
         />
         {memberCount === undefined && props.isLoading ? (
           <Shimmer height={10} width={normalize(25)} />
         ) : (
-          <Text style={styles.domainMember(shouldDisplay)}>{memberCount}</Text>
+          <Text
+            style={[
+              styles.domainMember(shouldDisplay),
+              isPreview ? {fontSize: normalizeFontSize(9)} : {}
+            ]}>
+            {memberCount}
+          </Text>
         )}
-        <Text style={styles.domainMember(shouldDisplay)}> Members</Text>
+        <Text
+          style={[
+            styles.domainMember(shouldDisplay),
+            isPreview ? {fontSize: normalizeFontSize(9)} : {}
+          ]}>
+          {' '}
+          Members
+        </Text>
       </View>
       {props?.initialData?.memberCount === undefined && props.isLoading ? (
         <Shimmer height={10} width={normalize(60)} />
       ) : (
         isFollow &&
         !hideSeeMember && (
           <Pressable onPress={handlePress} style={{backgroundColor: COLORS.transparent}}>
-            <Text style={styles.seeMemberText(followType)} numberOfLines={1} ellipsizeMode="tail">
+            <Text
+              style={[
+                styles.seeMemberText(followType),
+                isPreview ? {fontSize: normalizeFontSize(9)} : {}
+              ]}
+              numberOfLines={1}
+              ellipsizeMode="tail">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, consider moving the conditional styling based on the `isPreview` prop to the respective style functions (`styles.domainMember` and `styles.seeMemberText`). This will make the component cleaner and more maintainable.
```diff
-          width={isPreview ? 9 : 16}
-          height={isPreview ? 9 : 16}
+          style={styles.iconSize(isPreview)}

-          style={[
-            styles.domainMember(shouldDisplay),
-            isPreview ? {fontSize: normalizeFontSize(9)} : {}
-          ]}

-          style={[
-            styles.domainMember(shouldDisplay),
-            isPreview ? {fontSize: normalizeFontSize(9)} : {}
-          ]}

-          style={[
-            styles.seeMemberText(followType),
-            isPreview ? {fontSize: normalizeFontSize(9)} : {}
-          ]}
+          style={styles.domainMember(shouldDisplay, isPreview)}
+          style={styles.domainMember(shouldDisplay, isPreview)}
+          style={styles.seeMemberText(followType, isPreview)}
```
In your styles file, you can modify the `domainMember` and `seeMemberText` functions like this:
```javascript
domainMember: (shouldDisplay, isPreview) => ({
  // existing styles...
  fontSize: isPreview ? normalizeFontSize(9) : existingFontSize,
}),
seeMemberText: (followType, isPreview) => ({
  // existing styles...
  fontSize: isPreview ? normalizeFontSize(9) : existingFontSize,
}),
iconSize: (isPreview) => ({
  width: isPreview ? 9 : 16,
  height: isPreview ? 9 : 16,
}),
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->